PokeAlarm/Manager.py

```diff
@@ -242,7 +242,18 @@ def handle_pokemon(self, pkmn):
                 return
         else:
             log.debug("Pokemon inside geofences was not checked because no geofences were set.")
-
+        
+        height = pkmn['height']
+        weight = pkmn['weight']
+        gender = pkmn['gender']
+        if gender != 'unkn':
+            if gender == 1:
```

Comment:
Can we reduce this down to:
 `gender = u'\u2642' if gender is 1 else u'\u2640' if gender is 2 else u'\u26b2' # male, female, neutral`